lua/nui/popup/init.lua

```diff
@@ -86,11 +86,13 @@ function Popup:init(options)
     layout_ready = false,
     loading = false,
     mounted = false,
+    anchor = options.anchor,
     win_enter = options.enter,
     win_options = options.win_options,
     win_config = {
       focusable = options.focusable,
       style = "minimal",
+      anchor = options.anchor,
```

Comment:
This will be reset to `NW` if there's a border, since the popup window is then relative to the border window